bin/test.py

```diff
@@ -0,0 +1,7 @@
+import sys
+import tempfile; tempfile.tempdir = '/tmp'
```

Comment:
This shouldn't be needed. Both `cgap-portal` and `fourfront` have this in their top-level `conftest.py` which addreses it in a more reliable way because it means whether you use `bin/test` or not, you'll still get the support:
```
def pytest_configure():
    # This adjustment is important to set the default choice of temporary filenames to a nice short name
    # because without it some of the filenames we generate end up being too long, and critical functionality
    # ends up failing. Some socket-related filenames, for example, seem to have length limits. -kmp 5-Jun-2020
    tempfile.tempdir = '/tmp'
```
I recommend porting `cgap-portal/conftest.py` or `fourfront/conftest.py` and ditching this entire file (`bin/test.py`) as simply confusing. It's here only for backward compatibility in those portal repos.